lib/web_ui/dev/environment.dart

```diff
@@ -168,6 +168,13 @@ class Environment {
         'goldens',
       ));
 
+  /// Path to the screenshots taken by iOS simulator.
+  io.Directory get webUiSimulatorScreenshotsDirectory =>
+      io.Directory(pathlib.join(
+        webUiDartToolDir.path,
+        'screenshots',
```

Comment:
The directory name should reflect that the screenshots are coming from the iOS Simulator.

Reply:
done